recipes/pdfium/all/conanfile.py

```diff
@@ -27,84 +33,83 @@ class PdfiumConan(ConanFile):
         "with_libjpeg": "libjpeg",
     }
 
-    exports_sources = "CMakeLists.txt"
-    generators = "cmake", "cmake_find_package", "pkg_config"
-    short_paths = True
-
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
         self.requires("freetype/2.13.0")
-        self.requires("icu/73.2")
+        self.requires("icu/74.1")
         self.requires("lcms/2.14")
         self.requires("openjpeg/2.5.0")
+        self.requires("zlib/[>=1.2.11 <2]")
         if self.options.with_libjpeg == "libjpeg":
             self.requires("libjpeg/9e")
         elif self.options.with_libjpeg == "libjpeg-turbo":
-            self.requires("libjpeg-turbo/2.1.5")
-
-    def build_requirements(self):
-        self.build_requires("pkgconf/1.9.5")
+            self.requires("libjpeg-turbo/3.0.1")
 
     def validate(self):
         if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 14)
+            check_min_cppstd(self, 14)
         minimum_compiler_versions = {
-            "gcc": 8,
-            "Visual Studio": 15,
+            "gcc": "8",
+            "msvc": "191",
+            "Visual Studio": "15",
```

Comment:
Is there no restriction on clang, is the package not compatible with it (Seems like it is based on build logs) or just missing info?

Reply:
It built fine with Clang 5.0 in Docker. Conan 2 itself failed to install in the old Clang 4.0 container. I guess leaving it unspecified is fine.

Reply:
It built fine with Clang 5.0 in Docker. Conan 2 itself failed to install in the old Clang 4.0 container. I guess leaving it unspecified is fine.